src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_org_creator.erl

```diff
@@ -103,7 +109,9 @@
            {add_acl, [{container, policy_groups}], [read], [{group, clients}]},
            {add_acl, [{container, cookbook_artifacts}], [read], [{group, clients}]},
            {add_acl, [{container, data}], [read], [{group, clients}]},
-           {add_acl, mk_tl(container, [cookbooks, environments, roles]), [read] , [{group, clients}]}
+           {add_acl, mk_tl(container, [cookbooks, environments, roles]), [read] , [{group, clients}]},
+           {add_acl, [{container, policies}], [read], [{group, clients}]},
+           {add_acl, [{container, cookbook_artifacts}], [read], [{group, clients}]}
```

Comment:
I'm a bit confused by this bit.  Doesn't this duplicate the lines above (where line 108==113 and line 110==114)?